src/web/server/htmlTemplate.ts

```diff
@@ -103,6 +103,34 @@ export const htmlTemplate = ({
 
     const twitterMetaTags = generateMetaTags(twitterData, 'name');
 
+    // Duplicated prefetch and preconnect tags from DCP:
+    const staticPrefetchUrls = [
+        `https://assets.guim.co.uk/`,
+        `https://i.guim.co.uk`,
+        `https://api.nextgen.guardianapps.co.uk`,
+        `https://hits-secure.theguardian.com`,
+        `//j.ophan.co.uk`,
+        `//ophan.theguardian.com`,
+        `//phar.gu-web.net`,
+        `//www.google-analytics.com`,
+        '//sb.scorecardresearch.com',
+    ];
+
+    const staticPreconnectUrls = [
+        `https://assets.guim.co.uk/`,
+        `https://i.guim.co.uk`,
+        `//interactive.guim.co.uk`,
+        `//www.google-analytics.com`,
+    ];
+
+    const prefetchTags = staticPrefetchUrls.map(
+        src => `<link rel="dns-prefetch" href="${src}">`,
+    );
+
+    const preconnectTags = staticPreconnectUrls.map(
+        src => `<link rel="preconnect" href="${src}">`,
+    );
```

Comment:
Does order matter here?

Reading [the docs](https://developer.mozilla.org/en-US/docs/Web/Performance/dns-prefetch), it seems to indicate the `preconnect` should happen first. It's not explicit though but maybe just to be safe?

Reply:
oh cool - i can do that. it just did it in the same order as on DCP